features/google/src/context/google-context.jsx

```diff
@@ -70,6 +75,9 @@ export const GoogleContextProvider = ({ children }) => {
                 setIsLoading,
                 runWithLoadingScreen,
                 setGlobalError,
+                handleBack,
+                handleReportSent,
```

Comment:
Unless I'm missing something (very likely), isn't this functionally equivalent to `setReportIsSent`?